docs/en/pact-properties-api.md

```diff
@@ -635,6 +635,21 @@ List / string / object contains
 
 Supported in either invariants or properties.
 
+### enumerate {#FEnumerate}
+
+```lisp
+(drop n xs)
```

Comment:
The code excerpt here is about `drop`, not `enumerate`.

Reply:
Good catch, this was accidentally committed in #1155